### PR TITLE
Social Buttons Added

### PR DESCRIPTION
**Pull request by jmaguero**
+Facebook
+Instagram
+0,25em margin between them.

### CODE REVIEW

File: styles/globals.css [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 @tailwind utilities;
 
+/* .hoverMessage:hover {
```
Comment (Woile) [inline styles/globals.css:7]: Do we need this comment?

File: styles/globals.css [hunk 2]
```diff
@@ -4,6 +4,10 @@
...
+
+}
+ */
 .place-center {
```
Comment (Woile) [inline styles/globals.css:11]: where is this class used?

File: pages/index.js [hunk 3]
```diff
@@ -25,11 +25,28 @@ export default function Home() {
             <p>Join our Whatsapp group</p>
             <div className="w-full text-center flex justify-center">
               <a
+                className="hoverMessage mx-1"
```
Comment (Woile) [inline pages/index.js:28]: where is `hoverMessage`?

**Comment by jmaguero** [comment]
Now it is done :D

**Pull request merged**